### PR TITLE
add check condition when instance undefined

### PR DESCRIPTION
**Pull request by bleachalon**
Fix the issue of getInstance() return undefined after change == to ===.
Change instance name to be more semantic.

### CODE REVIEW

File: src/firebase/CRUDServices/experimentServices.js [hunk 1]
```diff
@@ -7,17 +7,18 @@ export default class ExperimentServices {
...
     /**
      * creating singleton instance.
      * @return {ExperimentServices} instance
      */
     static getInstance = () => {
-      if (self.sieInstance === null) {
-        self.sieInstance = new ExperimentServices();
+      if (self.experimentServiceInstance === null ||
+          self.experimentServiceInstance === undefined) {
+        self.experimentServiceInstance = new ExperimentServices();
```
Comment (quanganhhoang) [inline src/firebase/CRUDServices/experimentServices.js:19]: can we do `if (!self.experimentServiceInstance)` here? I'm just curious

https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in

Comment (alecherryy) [reply src/firebase/CRUDServices/experimentServices.js:19]: Nit-picky thing: the file names do not match the class names. I.e. `experimentService` vs `ExperimentService`.

**Comment by alecherryy** [comment]
@bleachalon Nit-picky thing: the file names do not match the class names. I.e. `experimentService.js` vs `ExperimentService`. They should both be capitalized.

**Review by alecherryy — APPROVED** [review]
Thank you for working on this. If you could update those two things, it'd be great!

**Pull request merged**